etc/ha_healthchecker/ha_healthchecker.py.j2

```diff
@@ -25,9 +28,15 @@ CONFIG_LOCAL = {
 {% if use_test_account %}
     'github_repo': "{{ test_repo_name }}",
     'github_user_token': "{{ test_github_token }}",
+    'github_user_passwd': "{{ test_github_password }}",
+    'github_user_name': "{{ test_github_user_name }}",
+    'github_app_name': "{{ test_github_app_name }}"
 {% else %}
     'github_repo': 'theopenlab/openlab',
     'github_user_token': "{{ github_token }}",
+    'github_user_passwd': "{{ github_password }}",
+    'github_user_name': "theopenlab-ci",
```

Comment:
1. Why not use {{ github_username}} here instead of hard code?
2. I don't think we have the var 'github_password' anymore, we use github_token instead of password, right?

Reply:
Yeah, I'm very clear what is done here, and I know very clear not use the password here.  But we could get only 1 way to change app webhook. Github API doesn't provide an API to change it[1].. So we only can simulate a human user to login github and change the app webhook by a python crawlers

reference:
[1] https://developer.github.com/v3/apps/